application/src/main/java/org/togetherjava/tjbot/commands/CommandHandler.java

```diff
@@ -88,9 +116,11 @@ private void getArgumentsAndCommandByEvent(GenericComponentInteractionCreateEven
         Command command = commandMap.get(argsArray[0]);
 
         if (command != null) {
-            List<String> args = new ArrayList<>(Arrays.asList(argsArray));
-            args = args.subList(2, args.size());
-            onSucceed.accept(command, args);
+            List<String> fullList = new ArrayList<>(Arrays.asList(argsArray));
+
+            final List<String> listExcludingStart = fullList.subList(2, fullList.size());
```

Comment:
`fullList` tells me nothing about this `List<String>`, maybe this is more descriptive:

```suggestion
            List<String> args = new ArrayList<>(Arrays.asList(argsArray));

            final List<String> argsExcludingFirst = fullList.subList(2, fullList.size());
```